mibs/dell/MSFT-MIB

```diff
@@ -0,0 +1,18 @@
+MSFT-MIB DEFINITIONS ::= BEGIN
+ 
```

Comment:
Ditto

Reply:
Same answer as above. I'm willing to go an alternate route with the folder location, but I'd like to get your thoughts first.